homeworks/dankhazipeter/2_vars_datatypes/user_info.py

```diff
@@ -0,0 +1,84 @@
+# Adott dictionary
+user_info = {
+    "name": "Mike",
+    "age": 25,
+    "favourite_meals": [
+        "pizza",
+        "carbonara",
+        "sushi"
+    ],
+    "phone_contacts": {
+        "Mary": "+36701234567",
+        "Tim": "+36207654321",
+        "Tim2": "+36304567321",
+        "Jim": "+364005000"
+    }
+}
+
+# 1. Kérj be a felhasználótól 4 programozási nyelvet
+programming_languages = input(
+    "Adj meg 4 programozási nyelvet vesszővel elválasztva, szóköz nélkül: ")
+skills_list = programming_languages.split(",")
+
+# Ellenőrizzük, hogy pontosan 4 elemet kaptunk-e
+if len(skills_list) != 4:
```

Comment:
Nagyon szuper! Nem is kell az egész kódot indentálni mert a raise miatt úgysem fut le semmi ezután, ha nem 4 elemű a listád. A 8 órán tanulunk ezekről, de nagyon király hogy már most használod!

Reply:
köszönöm